docs/user-guide/examples.md

```diff
@@ -194,6 +190,42 @@ These variables have to be set on the machine/container which host Træfik.
 
 These variables are described [in this section](/configuration/acme/#provider).
 
+### DNS challenge with wildcard domains
+
+```toml
+[entryPoints]
+  [entryPoints.https]
+  address = ":443"
+    [entryPoints.https.tls]
+
+[acme]
+email = "test@traefik.io"
+storage = "acme.json"
+caServer = "http://172.18.0.1:4000/directory"
```

Comment:
You should maybe use `"https://acme-staging-v02.api.letsencrypt.org/directory` instead of `http://172.18.0.1:4000/directory`